modules/core/04-channel/keeper/upgrade.go

```diff
@@ -51,13 +51,12 @@ func (k Keeper) WriteUpgradeInitChannel(ctx sdk.Context, portID, channelID strin
 		panic(fmt.Sprintf("could not find existing channel when updating channel state in successful ChanUpgradeInit step, channelID: %s, portID: %s", channelID, portID))
 	}
 
-	channel.State = types.INITUPGRADE
 	channel.UpgradeSequence++
 
 	k.SetChannel(ctx, portID, channelID, channel)
 	k.SetUpgrade(ctx, portID, channelID, upgrade)
 
-	k.Logger(ctx).Info("channel state updated", "port-id", portID, "channel-id", channelID, "previous-state", types.OPEN.String(), "new-state", types.INITUPGRADE.String())
+	k.Logger(ctx).Info("channel state updated", "port-id", portID, "channel-id", channelID, "state", channel.State.String(), "upgrade-sequence", fmt.Sprintf("%d", channel.UpgradeSequence))
```

Comment:
maybe we can address these logs in a follow up later, channel state is technically not updated anymore, only the upgrade sequence is incremented

Reply:
might as well drop the `.String` on the enum but fine if we do all these (since they must be in many places) in another round